docs/tutorials/g3-algebra-of-space.ipynb

```diff
@@ -44,6 +44,7 @@
    "outputs": [],
    "source": [
     "import clifford as cf\n",
+    "from numpy import pi,e\n",
```

Comment:
I think this isn't needed - we fixed this problem in a different way, but because you're using your `master` branch to make PRs this is including the changes from your previous PR

Reply:
```suggestion
```

Reply:
what branch should i use to make pr?

Reply:
A new branch for every PR is normally how it's done